capstone-ui/src/components/Feed/Feed.jsx

```diff
@@ -42,6 +60,16 @@ export default function Feed({ username, profile, token }) {
     <>
       {feed ? (
         <>
+          <div id="overlay">
+            <div className="profile-details-wrapper">
+              {hoverUsername && (
```

Comment:
Similar to recs suggestion, I find that reading hoverUsername != null is more explicit and easier to follow.